cluster-autoscaler/simulator/predicates.go

```diff
@@ -349,11 +349,18 @@ func (p *PredicateChecker) CheckPredicates(pod *apiv1.Pod, predicateMetadata pre
 	return nil
 }
 
-func (p *PredicateChecker) addDebugInfo(predInfo PredicateInfo, nodeInfo *schedulernodeinfo.NodeInfo) string {
+func emptyString() string {
+	return ""
+}
+
+func (p *PredicateChecker) addDebugInfo(predInfo PredicateInfo, nodeInfo *schedulernodeinfo.NodeInfo) func() string {
```

Comment:
more proper name would be `getDebugInfo`

Reply:
changed, PTAL